tutorials/server/install-and-configure-samba/install-and-configure-samba.md

```diff
@@ -0,0 +1,113 @@
+---
+id: install-and-configure-samba
+summary: Install and configure Samba to share files across a network.
+categories: server
+tags: tutorial,installation,ubuntu,file server,samba
+difficulty: 1
+status: Published
+published: 2017-12-01
+
+---
+
+# Install and Configure Samba
+
+## Overview
+Duration: 1:00
+
+A Samba file server enables file sharing across different operating systems over a network.
```

Comment:
I think this should be expanded a little, perhaps to say why you may want to go through this tutorial, eg. share your files with Windows and macOS users, access your desktop files from your laptop, set up a file server...